frequently-asked-questions.md

```diff
@@ -177,24 +177,26 @@ It is very popular and a great language!
 
 Here's a non-exhaustive list of differences:
 
-- Elixir is dynamically typed, Gleam is statically typed. Elixir is developing a
-  novel, but it has not yet released.
+- Elixir is dynamically typed, Gleam is statically typed. Elixir is integrating a
+  gradual type system into the language, but it has no user facing features yet.
 - Elixir has a powerful macro system, Gleam has no metaprogramming features.
 - Elixir's compiler is written in Erlang and Elixir, Gleam's is written in Rust.
-- Elixir uses Ruby style syntax, Gleam has a C family style syntax.
+- Gleam has a more traditional C family style syntax.
```

Comment:
I am not sure how much I'd describe Elixir as Ruby style. A possible suggestion is to emphasize Gleam being more traditional / familiar / etc. Your call.

Reply:
Seems reasonable to me